contributing/patches/load_local_synthetic_data.patch

```diff
@@ -0,0 +1,234 @@
+diff --git a/apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResourceGroup.java b/apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResourceGroup.java
+index 352569da..2883d2ed 100644
+--- a/apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResourceGroup.java
++++ b/apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResourceGroup.java
+@@ -44,6 +44,32 @@ public enum StaticRifResourceGroup {
+ 
+   SAMPLE_U(StaticRifResource.SAMPLE_U_BENES, StaticRifResource.SAMPLE_U_CARRIER),
+ 
++  LOCAL_SYNTHETIC(
++      StaticRifResource.LOCAL_SYNTHETIC_BENEFICIARY_1999,
++      StaticRifResource.LOCAL_SYNTHETIC_BENEFICIARY_2000,
++      StaticRifResource.LOCAL_SYNTHETIC_BENEFICIARY_2014,
++      StaticRifResource.LOCAL_SYNTHETIC_CARRIER_1999_1999,
++      StaticRifResource.LOCAL_SYNTHETIC_CARRIER_1999_2000,
++      StaticRifResource.LOCAL_SYNTHETIC_CARRIER_1999_2001,
++      StaticRifResource.LOCAL_SYNTHETIC_CARRIER_2000_2000,
++      StaticRifResource.LOCAL_SYNTHETIC_CARRIER_2000_2001,
++      StaticRifResource.LOCAL_SYNTHETIC_CARRIER_2000_2002,
++      StaticRifResource.LOCAL_SYNTHETIC_CARRIER_2014_2014,
++      StaticRifResource.LOCAL_SYNTHETIC_CARRIER_2014_2015,
++      StaticRifResource.LOCAL_SYNTHETIC_CARRIER_2014_2016,
++      StaticRifResource.LOCAL_SYNTHETIC_INPATIENT_1999_1999,
++      StaticRifResource.LOCAL_SYNTHETIC_INPATIENT_1999_2000,
++      StaticRifResource.LOCAL_SYNTHETIC_INPATIENT_1999_2001,
++      StaticRifResource.LOCAL_SYNTHETIC_INPATIENT_2000_2000,
++      StaticRifResource.LOCAL_SYNTHETIC_INPATIENT_2000_2001,
++      StaticRifResource.LOCAL_SYNTHETIC_INPATIENT_2000_2002,
++      StaticRifResource.LOCAL_SYNTHETIC_INPATIENT_2014_2014,
++      StaticRifResource.LOCAL_SYNTHETIC_INPATIENT_2014_2015,
++      StaticRifResource.LOCAL_SYNTHETIC_INPATIENT_2014_2016,
++      StaticRifResource.LOCAL_SYNTHETIC_PDE_2014,
++      StaticRifResource.LOCAL_SYNTHETIC_PDE_2015,
++      StaticRifResource.LOCAL_SYNTHETIC_PDE_2016,
++      StaticRifResource.LOCAL_SYNTHETIC_OUTPATIENT_1999_1999,
++      StaticRifResource.LOCAL_SYNTHETIC_OUTPATIENT_2000_1999,
++      StaticRifResource.LOCAL_SYNTHETIC_OUTPATIENT_2001_1999,
++      StaticRifResource.LOCAL_SYNTHETIC_OUTPATIENT_2002_2000,
++      StaticRifResource.LOCAL_SYNTHETIC_OUTPATIENT_2014_2014,
++      StaticRifResource.LOCAL_SYNTHETIC_OUTPATIENT_2015_2014,
++      StaticRifResource.LOCAL_SYNTHETIC_OUTPATIENT_2016_2014),
++
+   SYNTHETIC_DATA(
+       StaticRifResource.SYNTHETIC_BENEFICIARY_1999,
+       StaticRifResource.SYNTHETIC_BENEFICIARY_2000,
+diff --git a/apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java b/apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java
```

Comment:
I get an error at this line when trying to apply this patch:
```
cd ../; git apply contributing/patches/load_local_synthetic_data.patch
error: corrupt patch at line 209
```

Reply:
Perhaps this branch just needs to get merged with the latest from master?

Reply:
Fixed.